charts/virtaitech-orion-vgpu/v1.0.0/questions.yml

```diff
@@ -0,0 +1,172 @@
+categories:
+- repository
+labels:
+  io.rancher.certified: partner
+questions:
+- variable: defaultImage
+  default: "true"
+  description: "Use default Orion image, change this with caution!"
+  label: Use Default Image
+  type: boolean
+  show_subquestion_if: false
+  group: "Container Images"
+  subquestions:
+  # cannot set server iamge repository
+  #- variable: server.image.repository
+  #  default: "virtaitech/orion-server"
+  #  description: "Orion server image name"
+  #  type: string
+  #  label: Server Image Name
+  - variable: server.image.version
+    default: "2.1"
```

Comment:
`appVersion` & `image.version` better using the same value.

Reply:
Changed default value `2.1` to `2.2`